index.html

```diff
@@ -106,47 +110,108 @@ <h1>About me</h1>
             Hello I’m a software developer! I can help you build a product ,
             feature or website Look through some of my work and experience! If
             you like what you see and have a project you need coded, don’t
-            hestiate to contact me.
+            hesitate to contact me.
           </p>
-          <div class="botton-about">
-             <button type="submit">Get My Resume</button>
-            </div>
-        
+          <div class="button-about">
+            <button type="submit">Get My Resume</button>
+          </div>
         </div>
-        <div class="lenguages-sp"> 
+        <div class="languages-sp">
           <div class="about-me">
-        <h4 class="titles2"><img src="images2/rectangle.png" alt="rectangulo1" />Language</h4>
+            <h4 class="titles2">
+              <img src="images2/rectangle.png" alt="rectangulo1" />Language
+            </h4>
             <ul>
-             <li>Javascript</li>
+              <li>Javascript</li>
               <li>Ruby</li>
               <li>Html</li>
               <li>Css</li>
             </ul>
           </div>
           <div class="about-me">
-          <h4 class="titles2"><img src="images2/rectangle2.png" alt="rectangulo1"/>Framworks</h4>
+            <h4 class="titles2">
+              <img src="images2/rectangle2.png" class="" alt="rectangulo1" />Framworks
+            </h4>
             <ul>
               <li>Bootstrap</li>
               <li>RubyonRails</li>
               <li>RSPec</li>
               <li>CopyBara</li>
               <li>Selenium</li>
             </ul>
-        </div>
+          </div>
 
           <div class="about-me">
-            <h4 class="titles2"><img src="images2/rectangle3.png" alt="rectangulo1"/>Skills</h4>
+            <h4 class="titles2">
+              <img src="images2/rectangle3.png" alt="rectangulo1" />Skills
+            </h4>
             <ul>
               <li>Codekit</li>
               <li>Github</li>
               <li>Codepen</li>
               <li>Gitlab</li>
               <li>Terminal</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <section class="form">
+        <div>
```

Comment:
- [ ] Currently, there's a horizontal scroll bar on your page. Kindly fix it. You can use [CSS Media Queries](https://www.w3schools.com/css/css3_mediaqueries.asp) to make your page responsive well. :writing_hand: 

https://user-images.githubusercontent.com/70483835/165421027-54cbfa1f-5136-4ac9-b051-89b0356b0b53.mp4